Source/JavaScriptCore/jit/CCallHelpers.h

```diff
@@ -980,6 +980,9 @@ class CCallHelpers : public AssemblyHelpers {
 
     void emitCTIThunkPrologue(bool returnAddressAlreadyTagged = false);
     void emitCTIThunkEpilogue();
+
+    void brk() { breakpoint(); }
```

Comment:
Though I personally like contractions, this is against WebKit style rules.  I presume you did this because there is some namespace collision.  Can you provide details of how?  It is not clear.

Also, does anyone emit / use this right now?  I don't see any code reference to this other than in AirOpcodes.

Reply:
I think it is mostly used for debugging. I will remove it